File: androidApp/src/main/java/com/craftie/android/presentation/discovery/DiscoveryUiData.kt
package com.craftie.android.presentation.discovery

import com.craftie.data.model.Beer
import com.craftie.data.model.Brewery
import com.craftie.data.model.Province

data class DiscoveryUiData(
    val breweries: List<Brewery>,
    val beers: List<Beer>,
    val provinces: List<Province>
)

File: iosApp/iosApp/presentatioon/discovery/DiscoveryUiData.swift
//
//  DiscoveryUiData.swift
//  iosApp
//
//  Created by Ian  Arbuckle on 06/06/2021.
//  Copyright © 2021 orgName. All rights reserved.
//

import Foundation
import shared

struct DiscoveryUiData {
    var beers: [Beer]
    var breweries: [Brewery]
    var featuredBeer: Beer
    var provinces: [Province]
    var filteredBeersByDate: [Beer]
}

struct ProvinceLocal {
    var imageUrl: String
    var name: String
}
